src/mitol/google_sheets_refunds/api.py

```diff
@@ -80,17 +80,9 @@ def process_row(self, row_index, row_data):
                 message=None,
             )
         result_type, message = self.hook.refunds_process_request(refund_request)
-        if result_type != ResultType.PROCESSED:
-            # TODO: rollback the surrounding transaction
-            return RowResult(
-                row_index=row_index,
-                row_db_record=refund_request,
-                row_object=None,
-                result_type=ResultType.PROCESSED,
-                message=message,
-            )
-        refund_request.date_completed = now_in_utc()
-        refund_request.save()
+        if result_type is ResultType.PROCESSED:
```

Comment:
Should this really be `is`? it's a string comparison

Reply:
fixed